src/services/billing-services/list-payment-history-service.js

```diff
@@ -110,7 +118,7 @@ const adaptPaymentHistoryForRegularAccounts = (httpResponse) => {
       invoiceNumber: {
         content: card.billId
       },
-      disabled: !card.billId,
+      disabled: true,
```

Comment:
```suggestion
      disabled: true,
```
Recomendo analisar para caso realmente seja todos disabled, talvez seja o caso de alterar no ponto de utilização.